crates/spidermonkey-embedding-splicer/src/bindgen.rs

```diff
@@ -429,7 +429,7 @@ impl JsBindgen<'_> {
                                 // can be used when an explicit export is not
                                 // defined by the guest content.
                                 if iface_name == "incoming-handler"
-                                    || name == "wasi:http/incoming-handler@0.2.0"
+                                    || name.starts_with("wasi:http/incoming-handler@0.2")
```

Comment:
This would also match `0.20` - I think we need "0.2." as the prefix.

Reply:
ah yeah - Thanks! updated.

Reply:
You pushed a fix in the other occurrence but not at this spot

Reply:
oh my bad - I forgot to save the file. fixed it now